### ISSUE TITLE
Data split for training, testing and validation

### ISSUE THREAD

**Issue by ChataingT** [BelloneLab/lisbet#13]
Hi, 

I found those four parameters in the command line interface but their purpose is not clear to me.
    parser.add_argument(
        "--train_sample", type=float, help="Fraction of samples from the training set"
    )
    parser.add_argument(
        "--dev_sample", type=float, help="Fraction of samples from the dev set"
    )
    parser.add_argument(
        "--dev_ratio",
        type=float,
        help="Fraction of the training set to held out for hyper-parameters tuning",
    )
    parser.add_argument(
        "--test_ratio",
        type=float,
        help="""Fraction of the training set to held out for testing,
        if a test set is not available in the dataset""",
    )
    
   In my case, I already have a validation test on the side. I only need the classical train-test-split.
   Can you explain them to me ?
